tests/cpp/discrete_distribution.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <random>
+
+// std::mt19937 gives same output on different platforms
+// std::discrete_distribution is a platfrom specific
```

Comment:
```suggestion
// std::discrete_distribution is platform specific
```
`specific` is an adjective